### ISSUE TITLE
:analysis :symbols + :aliased-namespace-symbol linter gives false positive in quoted symbol

### ISSUE THREAD

**Issue by borkdude** [clj-kondo/clj-kondo#2039]
Repro:

```
$ clj -M:clj-kondo/dev --lint src/datomic.clj --config '{:analysis {:symbols true} :linters {:aliased-namespace-symbol {:level :warning}}}'
src/datomic.clj:8:26: warning: An alias is defined for datomic.api: d
```

``` clojure
(ns datomic
  (:require [datomic.api :as d]))

;; datomic.api/entid

(def foo (d/q '[:find [?artist-name ...]
                :in $ ?country [?reference ...]
                :where [(datomic.api/entid $ ?country) ?country-id]
                [?artist :artist/name ?artist-name]
                [?artist ?reference ?country-id]]))

```

cc @lassemaatta @ericdallo

**Comment by borkdude** [comment]
Should be good now! @lassemaatta @ericdallo
